setup.cfg

```diff
@@ -80,9 +80,9 @@ docstring-convention = numpy
 profile=black
 src_paths=interpax,test
 
-[options.data_files]
-* =
-  *.txt
+# [options.data_files]
```

Comment:
I think you can just delete these lines rather than comment them out, I think they were left over from a template for another project

Reply:
great, deleted them!